appcenter-analytics/Analytics.d.ts

```diff
@@ -0,0 +1,6 @@
+export function trackEvent(
+  eventName: string,
+  properties?: { [name: string]: string | number | boolean }
+): Promise<string | null>;
+export function isEnabled(): Promise<boolean | string>;
+export function setEnabled(enabled: boolean): Promise<string | null>;
```

Comment:
The promise does not return a string.

Reply:
Would you prefer setting it to an empty return then `: Promise` or `: Promise<null>`? Returning null seems like an odd thing to specify.

Reply:
I am not familiar with Promise typescript, but the type is "void", if Promise without type works, then yes we should use it.

Reply:
`Promise<void>` will do the trick 😄